cpp/examples/ide_secir.cpp

```diff
@@ -105,7 +115,9 @@ int main()
     sim.advance(tmax);
 
     auto interpolated_results = mio::interpolate_simulation_result(sim.get_result(), dt / 2);
-    interpolated_results.print_table({"S", "E", "C", "I", "H", "U", "R", "D "}, 16, 8);
+
+    interpolated_results.print_table({"S1", "E1", "C1", "I1", "H1", "U1", "R1", "D1 "}, 16, 8);
```

Comment:
Did you add the numbers because there is only one age group in this example? If so, I would remove them here because I find it confusing if there is only one age group. This also holds above for smoothcos1 etc. above.